### PR TITLE
Fixed a lot of Errors

### PR DESCRIPTION
**Pull request by Scriptkiddi**
Addressed most of the opened issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,31 +14,31 @@ See the [nginx doc](https://nginx.org/en/docs/http/ngx_http_core_module.html) fo
...
-| `nginx.ssl_prefer_server_ciphers`          | :heavy_multiplication_x:       | `on`|
-| `nginx.log_format`          | :heavy_multiplication_x:       | See defaults|
-| `nginx.access_log`          | :heavy_multiplication_x:       | `/var/log/nginx/access.log`|
-| `nginx.error_log`          | :heavy_multiplication_x:       | `/var/log/nginx/error.log`|
-| `nginx.gzip`          | :heavy_multiplication_x:       | `on`|
-| `nginx.gzip_disable`          | :heavy_multiplication_x:       | `msie6`|
-| `nginx.gzip_types`          | :heavy_multiplication_x:       | See defaults.|
+| `nginx_user`          | :heavy_multiplication_x:       | `www-data`          |
+| `nginx_worker_processes`          | :heavy_multiplication_x:       | `auto`  |
+| `nginx_pid`          | :heavy_multiplication_x:       | `/run/nginx_pid`|
```
Comment (dasJ) [inline README.md]: The file is still called `nginx.pid`

**Review by dasJ — DISMISSED** [review]
Wow, that was surprisingly fast. Thank you! Just this one issue, and I'm ready to merge

File: README.md (hunk 1 above)
Comment (Scriptkiddi) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -14,31 +14,31 @@ See the [nginx doc](https://nginx.org/en/docs/http/ngx_http_core_module.html) fo
...
-| `nginx.ssl_prefer_server_ciphers`          | :heavy_multiplication_x:       | `on`|
-| `nginx.log_format`          | :heavy_multiplication_x:       | See defaults|
-| `nginx.access_log`          | :heavy_multiplication_x:       | `/var/log/nginx/access.log`|
-| `nginx.error_log`          | :heavy_multiplication_x:       | `/var/log/nginx/error.log`|
-| `nginx.gzip`          | :heavy_multiplication_x:       | `on`|
-| `nginx.gzip_disable`          | :heavy_multiplication_x:       | `msie6`|
-| `nginx.gzip_types`          | :heavy_multiplication_x:       | See defaults.|
+| `nginx_user`          | :heavy_multiplication_x:       | `www-data`          |
+| `nginx_worker_processes`          | :heavy_multiplication_x:       | `auto`  |
+| `nginx_pid`          | :heavy_multiplication_x:       | `/run/nginx.:pid`|
```
Comment (SlothOfAnarchy) [inline README.md]: is the colon intended?

File: README.md [hunk 3]
```diff
@@ -50,6 +50,16 @@ See the [nginx doc](https://nginx.org/en/docs/http/ngx_http_core_module.html) fo
 | `served_domains.index`          | :heavy_check_mark:       |          | For which index files should nginx look|
 | `served_domains.locations.condition`          | :heavy_check_mark:       |          | The condition under which this locations block is called|
 | `served_domains.locations.content`          | :heavy_check_mark:       |          | Content of the locations block|
+| `served_domains.locations.ignore_access`          | :heavy_multiplication_x:       |          | Ignore the default acces behaviour|
```
Comment (SlothOfAnarchy) [inline README.md]: access

Comment (Scriptkiddi) [reply README.md]: done

**Comment by Scriptkiddi** [comment]
rereview

**Review by dasJ — APPROVED** [review]

**Pull request merged**